.github/workflows/release.yml

```diff
@@ -0,0 +1,26 @@
+name: release
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: Version to release
+        required: true
+      force:
+        description: Force a release even when there are release-blockers (optional)
+        required: false
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    name: "Release a new version"
```

Comment:
@NisanthanNanthakumar nit: should we use string quotations here?

Reply:
@scefali Did you mean single quotes? Prettier converts it to double quotes.